### DISCUSSION TITLE
What is the max size of csv file the "data openagent" can take ?

### DISCUSSION THREAD

**Discussion by pangjac** [xlang-ai/OpenAgents#102] [Q&A]
Dear community, 
By uploading my customized csv, I can ask data agent to analyze it. 
For the demo purpose, where can I find the max size limit of the csv file ? Is it restricted by Langchain or OpenAI API ?

Thanks for the hint!

**Comment by BlankCheng** [comment]
Sorry for missing the discussion. Hi, the max file size is 16MB. You can reset it in `frontend/utils/app/upload.ts` if you are hosting on the local machine.
